app/assets/stylesheets/sass/components/_notes.scss

```diff
@@ -0,0 +1,71 @@
+.note__container {
+  $note-background: $light-gray;
+  $base-border-radius: 3px !default;
+  $base-spacing: 1.5em !default;
+  $base-font-color: $dark-gray !default;
+  $note-gutter: 1.4em;
+  $appointment-date-padding: 0.7em;
+  $appointment-date-width: 6em;
+  $note-date-padding: 0.7em;
+  $note-date-width: 6em;
+  $note-color: $base-font-color;
+  $note-detail-color: transparentize($note-color, 0.5);
+  $note-date-vert-alignment: top;
+
+  border-bottom: 1px solid transparentize($note-color, 0.9);
+  display: table;
+  margin-bottom: $base-spacing;
+  padding-bottom: 1em;
+  width: 100%;
+
+  .note__date,
```

Comment:
Selector `note__date` should be written in lowercase with hyphens<br>Selector `note__content` should be written in lowercase with hyphens